### PR TITLE
Create capture_url plugin

### PR DESCRIPTION
**Pull request by JorgeVV**
A plugin to generate screenshots from any website. See README.md

### CODE REVIEW

**Review by olanod — CHANGES_REQUESTED** [review]
Awesome! looks quite good, just the minor cosmetic changes.

File: plugins.json [hunk 1]
```diff
@@ -1 +1,4 @@
-[{"type":"native","name":"blockchain", "prefix": "_vln"}]
+[
+  { "type": "native", "name": "blockchain", "prefix": "_vln" },
+  { "type": "native", "name": "capture_url", "prefix": "capture_url" }
```
[the remark this answers is not in the document]
Comment (JorgeVV) [reply plugins.json]: Got it!

**Comment by olanod** [comment]
Do you think we can do something about the webdriver server dependency with the failing tests?

**Comment by JorgeVV** [comment]
Yes. I was thinking about what we last talked about: setting up Docker. I haven't got time and I'll be working on this issue in the next couple of day.

**Comment by JorgeVV** [comment]
@olanod Finally 😄

**Review by olanod — COMMENTED** [review]
Changes look good :)  
Sorry if its late but I was thinking of a slight variation of the API that might be more ergonomic and then play better with the template rendering plugin.
Since returning a capture is actually fetching data, I think a `GET` request could work better here with the input data being just two query parameters, i.e. `?url=<URL>&d=<DIMENSIONS>`(dimensions can implement `FromStr` and then you just `.parse()` it), also comes with the added benefit of being easily cached by a CDN for example and we could shave off the _serde_ dependency.
I would add more details in the other PR but the idea is to leverage the conveniences of the HTTP protocol to generate the template with the other endpoint and automatically redirect here to get the capture all in one go, it could even work with a simple form without any extra code on the client side(no JS at all actually).

File: plugins/capture_url/src/lib.rs [hunk 2]
```diff
@@ -134,7 +135,7 @@ async fn handler(req: Request) -> Response {
         .unwrap_or_else(|| (&Cmd::Unknown, vec![]));
 
     match (action, req.method()) {
-        (Cmd::Capture, Method::Post) => capture_handler(req).await,
+        (Cmd::Capture, Method::Get) => capture_handler(req).await,
```
Comment (olanod) [inline plugins/capture_url/src/lib.rs:138]: Only change I can think is that this plugin maybe doesn't need the router dependency as it only does one thing, so just handlign get requests that come to its root should be good. That way the URL also becomes simpler as you would only do `curl http://api.valiu/capture-url?...`

**Review by olanod — APPROVED** [review]

File: plugins/capture_url/src/lib.rs (hunk 2 above)
Comment (JorgeVV) [reply plugins/capture_url/src/lib.rs]: Agreed and implemented 😄

**Pull request merged**